### ISSUE TITLE
RevelFramework Master Plan

### ISSUE THREAD

**Issue by pedromorgan** [revel/www#4]
The intention is to make revel eat is own dog food, ie revel uses revel.

##  Current www 
- At present the "web site" is at http://revel.github.io/ 
- and this is a jekyll/liquid site 
- code here https://github.com/revel/revel.github.io
- Thanks to gh, jekyll, markdown etc and it works fantastic
- any push to master rebuilds the whole site (unless a bug)
- and travis check its (after a strange bug >> )

## Content

- The manual, tutorial etc are in markdown (+gh.flavour)
- code here https://github.com/revel/revel.github.io
- each directory has text files in *.md eg manual/*
- each *.md file has a jekyll front matter eg layout: quickstart
- the layout_file containetains the section navigation
- the layoutfile includes_ the main_layout.html

**Comment by notzippy** [comment]
This might be best written up as an RFC under revel/rfc

**Comment by pedromorgan** [comment]
Good idea... RFC on the way

**Comment by pedromorgan** [comment]
ok changed plan.. I want an issues and the plan and stuff here..

# Pete tries to explains again, sorry guys.. welsh..

The idea in my head is to move some of the documentation from a "revel.gh.io/foo/bar" into their respecite repos notable for
- Modules
- Examples

case example is the 

## Jobs module

so docs at https://github.com/revel/revel.github.io/blob/master/modules/jobs.md
godoc at  https://godoc.org/github.com/revel/modules/jobs/app/jobs
and README in source = https://github.com/revel/modules/tree/master/jobs

So basically every example/ and module would be on rf.com
 - detected as they are added and appear in menu
- the modules+examples etc docs are self contained within the module..

We may even need some meta data suchs as
- config keys
- and varioous supervilous

- module+jobs  end up in README.md in the
